src/custom/state/orders/updater.ts

```diff
@@ -128,6 +133,73 @@ const constructGetLogsRetry = (provider: Web3Provider) => {
   return getLogsRetry
 }
 
+export function EventUpdaterApiOnly(): null {
+  const { chainId, library } = useActiveWeb3React()
+
+  const pending = usePendingOrders({ chainId })
+  const fulfillOrdersBatch = useFulfillOrdersBatch()
+
+  useEffect(() => {
+    let interval: NodeJS.Timeout | null = null
+    console.log('EventUpdaterApiOnly::useEffect')
+
+    if (!chainId || pending.length <= 0) {
+      console.log('EventUpdaterApiOnly::exiting', chainId, pending.length)
+
+      return
+    }
+
+    async function update(): Promise<void> {
+      console.log('EventUpdaterApiOnly::calling update')
+
+      const lastBlockNumber = await library?.getBlockNumber()
+      if (!chainId || !lastBlockNumber) {
+        console.log('EventUpdaterApiOnly::update, exiting', chainId, lastBlockNumber)
+
+        return
+      }
+      const orders = await Promise.all(
+        pending.map<Promise<[Order, OrderMetaData | null]>>(async storeOrder => {
+          const apiOrder = await getOrder(chainId, storeOrder.id)
```

Comment:
i saw this name (`apiOrder`) also in the other code, but it feels weird. I would expect that to hold an API to a order API, not an actual order. 

Later you use `orderFromApi` that sounds better.